.github/workflows/github_actions.yml

```diff
@@ -136,7 +136,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ macos-10.15, ubuntu-latest ]
+        os: [ macos-10.15, ubuntu-18.04 ]
```

Comment:
I suspect ubuntu-18.04 will not be available soon, please check the EOL dates when you have a chance.